keyboards/mechlovin/olly/jf/rev1/info.json

```
@@ -0,0 +1,663 @@
+{
+    "keyboard_name": "Olly JF Rev1",
+    "usb": {
+        "pid": "0xD180",
+        "device_version": "0.0.1"
+    },
```

migrated from `rules.mk`
```suggestion
    },
    "matrix_pins": {
        "custom_lite": true
    },
```